src/PresentationalComponents/shared/FormButtons.js

```diff
@@ -1,10 +1,32 @@
 import React from 'react';
 import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
-import FormSpy from '@data-driven-forms/react-form-renderer/form-spy';
 import PropTypes from 'prop-types';
 import { Button } from '@patternfly/react-core';
 import { isEmpty } from 'lodash';
 import { ActionGroup } from '@patternfly/react-core';
+import { FormSpy } from '@data-driven-forms/react-form-renderer';
+import './form-buttons.scss';
+
+const renderActionGroup = (noChanges, reset) => (
```

Comment:
Does this have to be a function? Could we make it a regular component?

Reply:
It's not needed anymore, forgot to remove it